config/eslint/eslintConfig.js

```diff
@@ -6,6 +6,10 @@ const baseConfig = {
   extends: orderImports
     ? [coreConfig, require.resolve('./importOrderConfig')]
     : coreConfig,
+  rules: {
+    'react/jsx-uses-react': 'off',
+    'react/react-in-jsx-scope': 'off',
```

Comment:
Currently disabling these rules inside sku itself, but we should probably lift this into `eslint-config-seek` once these changes have stabilised.